config/pitest-suppressions/pitest-coding-1-suppressions.xml

```diff
@@ -1,3 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressedMutations>
+  <mutation unstable="false">
+    <sourceFile>UnusedTryResourceShouldBeUnnamedCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.coding.UnusedTryResourceShouldBeUnnamedCheck</mutatedClass>
+    <mutatedMethod>beginTree</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
+    <description>Removed assignment to member variable nestingLevel</description>
+    <lineContent>nestingLevel = 0;</lineContent>
```

Comment:
Sorry, no leaks of test coverage.
We spent crazy amount of hours to reach good state, no excuses.